components/brave_rewards/browser/rewards_p3a.cc

```diff
@@ -0,0 +1,85 @@
+/* Copyright 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/brave_rewards/browser/rewards_p3a.h"
+
+#include "base/metrics/histogram_macros.h"
+
+namespace brave_rewards {
+
+void RecordWalletBalanceP3A(bool wallet_created, size_t b) {
+  int answer = 0;
+  if (wallet_created) {
+    if (b < 10) {
+      answer = 1;
+    } else if (10 <= b && b < 50) {
+      answer = 2;
+    } else if (50 <= b) {
+      answer = 3;
+    }
+  }
+  UMA_HISTOGRAM_EXACT_LINEAR("Brave.Rewards.WalletBalance", answer, 3);
+}
+
+void RecordAutoContributionsState(AutoContributionsP3AState state,
```

Comment:
nit: please move them into a new line

Reply:
What do you mean? With this and the other feedbacks, are you wanting to see the arguments each on its own line?

Reply:
Basically, I can run the clang formatter on everything. Let me do that